cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/dao/intervals/IntervalEntity.java

```diff
@@ -45,8 +45,11 @@
     private String id;
 
     @ClusteringColumn(1)
```

Comment:
Better store all partition keys and clustering keys in the same order as in table definition.
Interval start date is not clustering column 1, it is 2.